### PR TITLE
Move boost serialization to RTNeuron

### PR DESCRIPTION
**Pull request by eile**

### CODE REVIEW

File: CMake/Collage.spec [hunk 1]
```diff
@@ -1,13 +1,13 @@
 Name:		Collage
-Version:	1.2.1
+Version:
```
Comment (tribal-tec) [inline CMake/Collage.spec]: empty now?

File: CMake/Collage.spec [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
+Version:	
 Release:	1%{?dist}
 Summary:	Cross-platform C++ network library
 
 Group:		Development/Libraries
 License:	LGPLv2
 URL:		http://www.libcollage.net/
 Source0:	http://www.equalizergraphics.com/downloads/%{name}-%{version}.tar.gz
-Patch0:		Collage-1.2.1-build-fix.patch
+Patch0:		Collage--build-fix.patch
```
Comment (tribal-tec) [inline CMake/Collage.spec]: no version?

File: CMake/Collage.spec (hunk 1 above)
Comment (eile) [inline CMake/Collage.spec]: no clue.. will check.

**Comment by hernando** [comment]
No version change?

File: tests/distributable.cpp [hunk 3]
```diff
@@ -32,6 +34,7 @@ lunchbox::Monitorb received(false);
 class CameraBase : public servus::Serializable
 {
 public:
+    virtual CameraBase() {}
```
Comment (hernando) [inline tests/distributable.cpp]: This was supposed to be a destructor, wasn't it?

File: CMakeLists.txt [hunk 4]
```diff
@@ -33,7 +33,7 @@ set(COLLAGE_LICENSE LGPL)
 set(COLLAGE_DEPENDENT_LIBRARIES Boost Lunchbox Pression)
 
 common_find_package(Boost REQUIRED COMPONENTS system regex date_time
-  serialization program_options thread)
+  program_options thread)
```
Comment (tribal-tec) [inline CMakeLists.txt]: remove libboost-serialization-dev from above as well

**Pull request merged**